### PR TITLE
Add CLI option to add database and server url

### LINKED ISSUE #160: Add option to add database and server url in CLI
### I'm submitting a 
- [ ] bug report.
- [x] feature request.

### Current Behaviour:

If we want to use a database file instead of using sqlite in memory database we can not give it as database url through cli.

### Expected Behaviour:

User will be able to give database and server url as argument in hydrus CLI
### Do you want to work on this issue?

Yes

### PR DESCRIPTION
**Pull request by vddesai1998**
Closes https://github.com/HTTP-APIs/hydrus/issues/160

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.5.2 and above.

### Description

Add option for setting database URL and server URL in hydrus CLI

### Test Logs

cli

### CODE REVIEW

**Comment by chrizandr** [comment]
I was gonna create an issue about this, I'm glad someone did. Also, rather than keeping it as a required parameter, it's better to keep it optional. If the user does not provide a server url or db connection, let the cli default to `localhost` and a temporary db, using `sqlite:///:memory:'`

File: cli.py [hunk 1]
```diff
@@ -19,21 +19,23 @@
...
+@click.option("--dburl", default="sqlite:///:memory:", help="Set database url", type=str)
 @click.option("--hydradoc", "-d", default="doc.jsonld",
                 help="Location to HydraDocumentation (JSON-LD) of server.",
                 type=click.File('r'))
 @click.option("--port", "-p", default=8080,
                 help="The port the app is hosted at.", type=int)
+@click.option("--serverurl", default= "http://localhost", help="Set server url", type=str)
 @click.argument("serve", required=True)
-def startserver(adduser, api, auth, hydradoc, port, serve):
+def startserver(adduser, api, auth, dburl, hydradoc, port, serverurl, serve):
```
Comment (chrizandr) [inline cli.py]: Keep as optional parameters.

**Comment by chrizandr** [comment]
Have you tried running the server to see if everything is working?

**Comment by vddesai1998** [comment]
Yes I have run the server with the arguments and without providing arguments

**Pull request merged**